tests/gnmi/ethcounter_ixia_test.cc

```diff
@@ -1333,13 +1334,13 @@ TEST_P(ExampleIxiaTestFixture, TestIPv6Pkts) {
 
   // Connect to TestTracker for test status
   generic_testbed->Environment().SetTestCaseID(
```

Comment:
No test case ids in any test, pls fix this

Reply:
All the test case ids are removed.

> No test case ids in any test, pls fix this

All the test case ids are removed.